model_analyzer/config/input/config_command_profile.py

```diff
@@ -604,19 +612,20 @@ def _preprocess_and_verify_arguments(self):
                     "client-protocol is 'grpc'. Must specify triton-grpc-endpoint "
                     "if connecting to already running server or change protocol using "
                     "--client-protocol.")
-        elif self.triton_docker_mounts:
+        elif self.triton_docker_mounts or self.triton_docker_labels:
             if self.triton_launch_mode == 'docker':
                 # Verify format
-                for volume_str in self.triton_docker_mounts:
-                    if volume_str.count(':') != 2:
-                        raise TritonModelAnalyzerException(
-                            "triton_docker_mounts needs to be a list of strings. Each string "
-                            " should be of the format <host path>:<container dest>:<access mode>"
-                        )
+                if self.triton_docker_mounts:
+                    for volume_str in self.triton_docker_mounts:
+                        if volume_str.count(':') != 2:
+                            raise TritonModelAnalyzerException(
+                                "triton_docker_mounts needs to be a list of strings. Each string "
+                                " should be of the format <host path>:<container dest>:<access mode>"
```

Comment:
Why require the access mode and not make it optional?

Reply:
It simplifies processing this argument and verifying the argument (check for 2 ':' in the string). I don't see a huge advantage in affording the user the flexibility to leave the mode unspecified and to use a default. Do you think this is worth it?

Reply:
Yeah, I agree it is fine in its current mode too. It was only a suggestion.